app/models/concerns/foreman_host_reports/host_extensions.rb

```diff
@@ -12,7 +12,7 @@ module HostExtensions
       scoped_search relation: :host_reports, on: :format, rename: :report_format, only_explicit: true, complete_value: { plain: 0, puppet: 1, ansible: 2 }
 
       def last_host_report_object
-        host_reports.order("#{HostReport.table_name}.reported_at DESC, #{HostReport.table_name}.id").limit(1)&.first
+        host_reports.reorder("#{HostReport.table_name}.id DESC").limit(1)&.first
```

Comment:
We use last report query only via IDs in other places as well as for legacy reports, this had a bug so I fixed it (ID was sorted ascending). This is more simple.